gen/dcos-config.yaml

```diff
@@ -1758,3 +1758,14 @@ package:
         dcos_node_private_ip = "$DCOS_NODE_PRIVATE_IP"
         # Global DC/OS Cluster ID.
         dcos_cluster_id = "$DCOS_CLUSTER_ID"
+  - path: /etc/telegraf/telegraf.d/all.conf
```

Comment:
Hi @urbanserj, 

The canonical way to do that is to add this configuration to telegraf.conf, rather than in its own separate file: https://github.com/dcos/dcos/blob/master/gen/dcos-config.yaml#L1313

Reply:
Got it, I'll move the configuration to `telegraf.conf`. Thank you!